### DISCUSSION TITLE
Unstable wifi AP connection

### DISCUSSION THREAD

**Discussion by Nate-Boyette** [nakhonthai/ESP32APRS_T-TWR#10] [General]
Is anyone else having trouble accessing their T-TWR via the wifi AP? When enabled the AP shows up very intermittently on my laptop and cellphone wifi scan. Its bad enough that i have not been able to successfully connect to that AP with any device

**Comment by lyusupov** [comment]
Yes. Firmware default settings does not seem to be good enough to operate Wi-Fi Access Point - both a Windows 11 PC and two of Android phones can not connect.

I had to use the T-TWR Plus encoder wheel with OLED UI for activating Wi-Fi STA mode in order to connect to my local router.
The Wi-Fi operations of the firmware work smooth afterwards.

**Comment by lyusupov** [comment]
recent video from @nakhonthai 
https://www.youtube.com/watch?v=e9tvUChsF4U

**Comment by captbd** [comment]
Watched the above video; tried with Android, iphone, windows computer.  Get as far as login page.  admin  admin are rejected, ignored

**Comment by lyusupov** [comment]
>  admin admin are rejected,

https://github.com/nakhonthai/ESP32APRS_T-TWR/issues/18#issuecomment-1886574522

**Comment by CcRup** [comment]
Running 0.4. Drops over my Android phone but OK with Win 10 laptop.
